### PR TITLE
Update doc/newshell with the information about the splitted command descriptions ##doc

### PR DESCRIPTION
**Pull request by officialcjunior**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

On #687, `cmd_descs.yaml` was partitioned into in smaller YAML files. This PR
makes very small changes to doc/newshell.md, so that the development procedure
aligns with the new method.

**Test plan**

- [ ] Review whether it is phrased correctly and change if necessary.

**Closing issues**

None.

### CODE REVIEW

File: doc/newshell.md [hunk 1]
```diff
@@ -81,13 +81,14 @@ hashtable, using their names as keys.
...
 would find all occurrences of the word "sky" in a binary. The first thing to
-do is to insert the `sky` command in
-[`librz/core/cmd_descs.yaml`](https://github.com/rizinorg/rizin/blob/9ce5003ca647cdfc181ac3c0f6206762ebb9e3e9/librz/core/cmd_descs.yaml). That
-file respects the same tree structure used when executing rizin and seeing
-its help, so it should be simple to see where to place it. If we want to
-place it under the `s` sub-tree, we just need to define a new descriptor
-under `subcommands`, with at least `name`, `cname`, `summary` and a list of
-`args` accepted by the command.
+do is to see where `sky` command could be added by reading
+[`librz/core/cmd_descs/cmd_descs.yaml`](https://github.com/rizinorg/rizin/blob/6d901ca8a2ba674e268957fa8c24b3484bfc3626/librz/core/cmd_descs/cmd_descs.yaml).
```
Comment (XVilka) [inline doc/newshell.md:85]: Probably worth mentioning it would be in `librz/core/cmd_descs/cmd_seek.yaml` since it starts from `s`.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: doc/newshell.md (hunk 1 above)
Comment (officialcjunior) [reply doc/newshell.md:85]:
```suggestion
[`librz/core/cmd_descs/cmd_descs.yaml`](https://github.com/rizinorg/rizin/blob/6d901ca8a2ba674e268957fa8c24b3484bfc3626/librz/core/cmd_descs/cmd_descs.yaml).
```

File: doc/newshell.md [hunk 2]
```diff
@@ -81,13 +81,15 @@ hashtable, using their names as keys.
...
+`sky` is `s` command's subcommand and they are splitted and placed inside the .YAML 
+file specified by the descriptor `subcommands` of the respective command. Since `sky`
+starts with an `s`, its subcommands would be in [`librz/core/cmd_descs/cmd_seek.yaml`](https://github.com/rizinorg/rizin/blob/6d901ca8a2ba674e268957fa8c24b3484bfc3626/librz/core/cmd_descs/cmd_seek.yaml).
+That file respects the same tree structure used when executing rizin and seeing its help,
+so it should be simple to see where to place it.If we want to place it under the
```
Comment (XVilka) [inline doc/newshell.md]: Missing space after dot.

File: doc/newshell.md [hunk 3]
```diff
@@ -81,13 +81,15 @@ hashtable, using their names as keys.
...
+`s` sub-tree, we just need to define a new descriptor under `subcommands`, with
```
Comment (ret2libc) [inline doc/newshell.md]: I believe there is no subcommands in the split file

Comment (officialcjunior) [reply doc/newshell.md]: Oh yeah, there isn't any.
How about now?

**Review by ret2libc — APPROVED** [review]
@XVilka please have another look as well.

**Review by XVilka — APPROVED** [review]

**Pull request merged**